response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -435,16 +434,11 @@ def link_sample_summary_to_collection_exercise(collection_exercise_id, sample_su
     return response.json()
 
 
-def get_collection_exercises_with_samples_by_survey_id(survey_id):
+def get_collection_exercises_by_survey_id(survey_id):
     logger.info("Retrieving collection exercise with samples", survey_id=survey_id)
```

Comment:
Is this method even needed anymore? All it does is logs and calls another method in the same controller

Reply:
Fixed. I've rewritten calls of this function to use the one nested inside.